app/src/components/modal/modal_deposit_withdraw/index.tsx

```diff
@@ -56,20 +63,32 @@ const WalletText = styled.div`
 `
 
 const DepositWithdrawButton = styled(Button)`
-  width: 100%;
-  margin-top: 24px;
+  flex: 1;
+`
+const ApproveButton = styled(ButtonStateful)`
+  flex: 1;
+  margin-right: 16px;
 `
 
-const Allowance = styled.div`
+const ExchangeDataItem = styled.div`
   display: flex;
-  margin-top: 20px;
-  padding: 16px 20px;
-  border-radius: 4px;
-  border: ${props => props.theme.borders.borderLineDisabled};
-  line-height: 20px;
+  justify-content: space-between;
+  width: 100%;
+  line-height: 16px;
```

Comment:
i learned from the very best, so this value should be passed through `props.theme.fonts.defaultLineHeight`